crates/provider/src/ext/trace.rs

```diff
@@ -119,6 +126,15 @@ where
         self.client().request("trace_transaction", (hash,)).await
     }
 
+    async fn trace_get(
+        &self,
+        hash: TxHash,
+        index: u64,
```

Comment:
since this is an index, I wonder if we should use usize here because I assume most of the time this value is derived from doing some vec.len() +- ops

Reply:
That's a good point, put usize in there.
I think it would be less convenient to put Index as type because then all calls to trace_get would require a .into() call from the user. It probably would be less convenient.